<launch>
#<arg name="model" default='$(find urdf_robot)/urdf/dynamic_robot.urdf' />
<arg name="gui" default="False" />
<param name="robot_description" textfile="$(arg model)" />
<param name="use_gui" value="$(arg gui)"/>
<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
<node name="InterpolGrasper" pkg="urdf_robot" type="InterpolGrasper" />
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find urdf_robot)/rviz/pose_inter.rviz" required="true" />
</launch>
